### PR TITLE
Setup Android Project

### PR DESCRIPTION
**Pull request by sdsantos**

### CODE REVIEW

**Comment by gnarea** [comment]
Nice! Is this ready to be reviewed or should I hold off?

**Comment by sdsantos** [comment]
Ready, I'll do one for each step.

**Comment by gnarea** [comment]
Haha, sorry, I think I'm too eager -- I just realised I got an email notification because I'm watching the repo, not because you assigned it to me :smile:

**Comment by gnarea** [comment]
Oh cool, in that case I'll review it now

File: app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,35 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+android {
+  compileSdkVersion 29
+  buildToolsVersion "29.0.2"
+
+  defaultConfig {
+    applicationId "link.relaynet.courier"
```
Comment (gnarea) [inline app/build.gradle]: Can we please use `tech.relaycorp` as the prefix? E.g., `tech.relaycorp.courier`

File: app/src/androidTest/java/link/relaynet/courier/ExampleInstrumentedTest.kt [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package link.relaynet.courier
```
Comment (gnarea) [inline app/src/androidTest/java/link/relaynet/courier/ExampleInstrumentedTest.kt]: See comment above regarding package name

**Pull request merged**